samples/png.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 The Amber Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "samples/png.h"
+
+#include <cassert>
+
+#include "amber/result.h"
+#include "amber/value.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wweak-vtables"
+#include "third_party/lodepng/lodepng.h"
+#pragma clang diagnostic pop
+
+namespace png {
+
+namespace {
+
+unsigned char byte0(uint32_t word) {
```

Comment:
Normally I'd ask for uint8_t here. But lodepng wants a vector of unsigned char.  So I'll let this pass.

Reply:
ack